django_app/redbox_app/redbox_core/models.py

```diff
@@ -614,32 +612,8 @@ def file_type(self) -> str:
         return name.split(".")[-1]
 
     @property
-    def url(self) -> URL | None:
-        #  In dev environment, get pre-signed url from minio
-        if settings.ENVIRONMENT.uses_minio:
```

Comment:
Presuming you no longer need this in dev?

Reply:
we dont seem to - integration tests and unit tests look fine